cypress/integration/utils/index.ts

```diff
@@ -99,15 +118,28 @@ export function runTestsForDevices({
   devices.forEach(d => {
     // Skip tests that require login if username and password not found
     const skipForLogin = d.loggedIn && (!username || !password);
-
     if (skip || skipForLogin) {
-      describe.skip(d.description, () => {
-        runTests({ ...d, currentPage });
-      });
+      describe.skip(
+        d.description,
+        {
+          viewportHeight: deviceDimensions[d.device].height,
+          viewportWidth: deviceDimensions[d.device].width,
+        },
```

Comment:
If you made the type of `deviceDimensions` `Record<Cypress.ViewportPresent, Cypress.Viewport>` you could just do this here
```suggestion
       deviceDimensions[d.device],
```